src/lib/config/env.ts

```diff
@@ -108,6 +108,11 @@ export function hubspotContactConfigFromENV(): HubspotContactConfig {
 export function mpacConfigFromENV(): MpacConfig {
   return {
     ignoredEmails: new Set((optional('IGNORED_EMAILS')?.split(',') ?? []).map((e) => e.toLowerCase())),
+    emailMappings: optional('EMAIL_MAPPINGS')?.split(',').reduce((p: any, c: string) => {
```

Comment:
I guess `p` is of type `Record<string, string>` right? We should avoid `any` if possible.

Reply:
done